xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -74,7 +74,7 @@ void CGUIWindowPVRGuideBase::Init()
     epgGridContainer->GoToNow();
   }
 
-  if (!m_refreshTimelineItemsThread)
```

Comment:
Yes, the check is replaced by something that not only kicks in during very first open of the window, but also after the window controls have been replaced when doing a skin reload. Much cleaner now - and fixing the bug. 

The member is used at other places in this file.